docs/user/addons.rst

```diff
@@ -46,8 +46,323 @@ Individual configuration options for each addon are available in :guilabel:`Sett
+                    },
+                    {
+                        "username": "agj"
+                    },
+                    {
+                        "username": "stsewd"
+                    }
+                ],
+                "versioning_scheme": "multiple_versions_with_translations"
+            },
+            "translations": []
+        },
+        "readthedocs": {
+            "analytics": {
+                "code": "UA-17997319-1"
+            }
+        },
+        "versions": {
+            "active": [
+                {
+                    "active": true,
+                    "aliases": [],
+                    "built": true,
+                    "downloads": {
+                        "epub": "https://docs.readthedocs.io/_/downloads/en/stable/epub/",
+                        "htmlzip": "https://docs.readthedocs.io/_/downloads/en/stable/htmlzip/"
+                    },
+                    "hidden": false,
+                    "id": 2604018,
+                    "identifier": "6db46a36ed3da98de658b50c66b458bbfa513a4e",
+                    "privacy_level": "public",
+                    "ref": "11.18.0",
+                    "slug": "stable",
+                    "type": "tag",
+                    "urls": {
+                        "dashboard": {
+                            "edit": "https://readthedocs.org/projects/docs/version/stable/edit/"
+                        },
+                        "documentation": "https://docs.readthedocs.io/en/stable/",
+                        "vcs": "https://github.com/readthedocs/readthedocs.org/tree/11.18.0/"
+                    },
+                    "verbose_name": "stable"
+                }
+            ],
+            "current": {
+                "active": true,
+                "aliases": [],
+                "built": true,
+                "downloads": {
+                    "epub": "https://docs.readthedocs.io/_/downloads/en/stable/epub/",
+                    "htmlzip": "https://docs.readthedocs.io/_/downloads/en/stable/htmlzip/"
+                },
+                "hidden": false,
+                "id": 2604018,
+                "identifier": "6db46a36ed3da98de658b50c66b458bbfa513a4e",
+                "privacy_level": "public",
+                "ref": "11.18.0",
+                "slug": "stable",
+                "type": "tag",
+                "urls": {
+                    "dashboard": {
+                        "edit": "https://readthedocs.org/projects/docs/version/stable/edit/"
+                    },
+                    "documentation": "https://docs.readthedocs.io/en/stable/",
+                    "vcs": "https://github.com/readthedocs/readthedocs.org/tree/11.18.0/"
+                },
+                "verbose_name": "stable"
+            }
+        }
+
+You can see a live example of this in our `Addons API response for these docs <https://docs.readthedocs.io/_/addons/?client-version=0.22.0&api-version=1&project-slug=docs&version-slug=stable>`_.
```

Comment:
This could also be a `.. tip::`

Reply:
I don't think a tip is really useful? I don't want to highlight this too visually, and it's the only content in the section that is expanded by default.

Reply:
It might be worth dropping this too, we already have an example above. Is there a reason there needs to be two examples? Maybe it's worth just bringing in this data response as the example above?